docs/docs/sdk-examples/python.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Python
+sidebar_label: NestJS
+---
+
+<table>
+    <tr>
+        <td width="100px">
+            <img width="100px" src="/docs/img/tech/python.svg" alt="phyton logo"/>
+        </td>
+        <td>
+            <a href="https://flask.palletsprojects.com/en/3.0.x/" target="_blank">Flask</a> is a lightweight and easy-to-use microframework for [Python](https://www.python.org/) web development.
+            Get started integrating authentication to your Flask API Application by checking out our <a href="https://github.com/zitadel/example-api-python3-flask" target="_blank">example-api-python3-flask</a> Example.
+        </td>
+    </tr>
+</table>
+
+
+### Resources
+- [Example App Repository](https://github.com/zitadel/example-api-python3-flask)
+- [API APP Step-By-Step Guide](/docs/examples/secure-api/python-flask)
+- [Frontend with backend API](https://github.com/zitadel/example-quote-generator-app): A simple web application using a React front-end and a Python back-end API, both secured using ZITADEL
+- [Fine-grained authorization](https://github.com/zitadel/example-fine-grained-authorization): Leverage actions, custom metadata, and claims for attribute-based access control
+
+### Flask SDK
+
+ZITADEL does not provide a Python Flask specific SDK.
+But you can integrate ZITADEL to your application by using any OIDC Library such as [angular-oauth2-oidc](https://github.com/manfredsteyer/angular-oauth2-oidc).
+
+Check out our [Example Application](/docs/sdk-examples/angular#example-application)
+
+
+### Go Examples
+
+What does the API Application Example include:
+- REST API Application secured with Spring Security and OAuth2
+- Public Endpoint: Accessible without authentication
+- Private Endpoint: Accessible with a token
+- Administrator Endpoint: Accessible with a token of a user with admin role
+
+[https://github.com/zitadel/example-api-python3-flask)
```

Comment:
```suggestion
[Example API App](https://github.com/zitadel/example-api-python3-flask)
```

Reply:
It seems the suggestion did not get applied, still broken